UnboundLib/Utils/LevelManager.cs

```diff
@@ -303,33 +292,18 @@ internal static void SpawnMap(string message)
                 var bestMatches = 0f;
 
                 // parse message
-                var formattedMessage = message.ToUpper()
-                    .Replace(" ", "_")
-                    .Replace("/", "");
-
+                var formattedMessage = message.ToUpper().Replace(" ", "_").Replace("/", "");
```

Comment:
Repeated calls like this are most readable on separate lines. Recommend reverting